code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/ConatainerBaseGenerator.xtend

```diff
@@ -0,0 +1,24 @@
+package de.fhdo.lemma.model_processing.code_generation.container_base
+
+import de.fhdo.lemma.model_processing.AbstractModelProcessor
+
+/**
+ * Main class of the container base generator implemented on the basis of LEMMA's model processing
+ * framework as an [AbstractModelProcessor].
```

Comment:
The `[...]` reference syntax does only work in Kotlin. Please remove the square brackets in all Javadoc comments.